ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/io/GroupValueTranscoder.java

```diff
@@ -1,96 +1,68 @@
 package org.ccci.idm.user.ldaptive.dao.io;
 
-import com.google.common.base.Strings;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Lists;
+import com.google.common.annotations.VisibleForTesting;
 import org.ccci.idm.user.Dn;
 import org.ccci.idm.user.Group;
-import org.ldaptive.DnParser;
-import org.ldaptive.LdapAttribute;
+import org.ccci.idm.user.ldaptive.dao.util.DnUtils;
 import org.ldaptive.io.AbstractStringValueTranscoder;
 
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
-import java.util.List;
 
 public class GroupValueTranscoder extends AbstractStringValueTranscoder<Group> {
-    private static final String delimiter = ",";
-    private static final String valueDelimiter = "=";
+    @Nonnull
+    private Dn baseDn = Dn.ROOT;
```

Comment:
it might be better to move this functionality to `LdaptiveUserDao` now. It served 2 roles, first to relativize groups which no longer happens. second was to provide a global restriction as to which groups could be manipulated.

the second responsibility is probably better off being enforced within the `LdaptiveUserDao` now